queries.sql

```diff
@@ -1 +1,41 @@
 /* Find all animals whose name ends in "mon".*/
```

Comment:
- [ ] Please add the semicolon `;` after every query. The semicolon tells that the query is completed. If you run a query in the terminal without a semicolon it will give not run.

Reply:
Done